src/client/runner/automation/playback/press/utils.js

```diff
@@ -69,3 +70,21 @@ export function getChar (key, shiftModified) {
     return key;
 }
 
+export function getDeepActiveElement (currentDocument) {
+    var doc                   = currentDocument || document;
```

Comment:
This function must be in `testcafe-hammerhead` actually, we need to get deep active element in [focus-blur sandbox here](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/event/focus-blur.js#L212) AFAIK. Strange, that `hammerhead` tests was not broken by https://github.com/DevExpress/testcafe-hammerhead/pull/470.